### PR TITLE
Bump dotnet-format

### PR DESCRIPTION
**Pull request by sungam3r**

### CODE REVIEW

File: src/Directory.Build.props [hunk 1]
```diff
@@ -15,6 +15,7 @@
     <DebugType>embedded</DebugType>
     <EmbedUntrackedSources>true</EmbedUntrackedSources>
     <EnableNETAnalyzers>true</EnableNETAnalyzers>
+    <NoWarn>$(NoWarn);1591</NoWarn> <!--TODO: temporary solution-->
```
Comment (Shane32) [inline src/Directory.Build.props:18]: With this change, let's remove ` -p:NoWarn=CS1591` from the github action workflow(s).

**Review by Shane32 — CHANGES_REQUESTED** [review]

File: src/Directory.Build.props (hunk 1 above)
Comment (sungam3r) [reply src/Directory.Build.props:18]: I expected this suggestion :)

Comment (sungam3r) [reply src/Directory.Build.props:18]: done

File: .github/workflows/test-code.yml [hunk 2]
```diff
@@ -30,22 +30,22 @@ jobs:
...
+      - name: Install dependencies
+        working-directory: src
+        run: dotnet restore
       - name: Check formatting
         if: ${{ startsWith(matrix.os, 'ubuntu') }}
         working-directory: src
         run: |
-          dotnet tool install -g dotnet-format
-          dotnet format --check -v diag || (echo "Run 'dotnet format' to fix formatting issues" && exit 1)
-      - name: Install dependencies
```
Comment (sungam3r) [inline .github/workflows/test-code.yml:39]: Some analyzers require already installed dependencies, so this step has been moved up.

**Comment by sungam3r** [comment]
изображение

File: src/GraphQL.DataLoader.Tests/Types/UserType.cs [hunk 3]
```diff
@@ -43,15 +43,10 @@ public UserType(IDataLoaderContextAccessor accessor, IOrdersStore orders)
...
-                            //var batchResults = await orders.GetItemsByOrderIdAsync(orderResults.Select(o => o.OrderId));
-                            //var allResults = orderResults.Select(o => batchResults[o.OrderId]);
-
-                            //flatten and return the results
-                            return allResults.SelectMany(x => x);
-                        });
+                        //without dataloader, this would be:
+                        //var batchResults = await orders.GetItemsByOrderIdAsync(orderResults.Select(o => o.OrderId));
+                        //var allResults = orderResults.Select(o => batchResults[o.OrderId]);
+                        return itemsLoader.LoadAsync(orderResults.Select(o => o.OrderId)).Then(allResults => allResults.SelectMany(x => x)); //flatten and return the results
```
Comment (Shane32) [inline src/GraphQL.DataLoader.Tests/Types/UserType.cs:49]:
```suggestion
                        return itemsLoader.LoadAsync(orderResults.Select(o => o.OrderId))
                            .Then(allResults => allResults.SelectMany(x => x)); //flatten and return the results
```

**Review by Shane32 — APPROVED** [review]

**Comment by Shane32** [comment]
I would rather that within Visual Studio we could see the 1591 warnings.

**Comment by Shane32** [comment]
I've got rid of a couple hundred 1591 warnings in the last couple weeks.  Hopefully I can blast through a bunch more here over the weekend.

**Comment by Shane32** [comment]
> I would rather that within Visual Studio we could see the 1591 warnings.

But it's fine for now.  We need to get through the bulk of them first anyway.  And that's just double clicking on every file and adding comments.

**Comment by sungam3r** [comment]
> I would rather that within Visual Studio we could see the 1591 warnings.

Just comment NoWarn in props file.

**Comment by Shane32** [comment]
I think I'll need your help for some of the xml comments.  I'm trying to go through everything I can and hopefully you can tackle the last few methods and classes I'm unfamiliar with.

**Pull request merged**